otel-worker-cli/src/commands/client/traces.rs

```diff
@@ -55,6 +57,13 @@ async fn handle_get(args: GetArgs) -> Result<()> {
 
 #[derive(clap::Args, Debug)]
 pub struct ListArgs {
+    /// Optional limit to the amount of traces to be returned
+    pub limit: Option<u32>,
+
+    /// Optional timestamp from which traces that occurred before it are returned.
+    /// Must be in RFC3339 format
+    pub time: Option<String>,
```

Comment:
Can't we change this to a typed object and then use serde's `with`? https://docs.rs/time/latest/time/serde/rfc3339/option/index.html

Reply:
Oh didn't notice this is actually clap and not serde. Good fix though :+1: